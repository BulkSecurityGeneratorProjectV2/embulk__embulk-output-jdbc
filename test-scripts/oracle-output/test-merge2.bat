sqlplus TEST_USER/XXXXXXXX@localhost/TESTDB @test-merge.sql
sqlplus TEST_USER/XXXXXXXX@localhost/TESTDB @test-merge-data.sql

CALL embulk run test-merge2.yml

sqlplus TEST_USER/XXXXXXXX@localhost/TESTDB @select.sql

echo "diff data/test-merge2_expected.txt data/temp.txt"
diff data/test-merge2_expected.txt data/temp.txt

IF "%ERRORLEVEL%" == "0" (ECHO "OK!") ELSE (ECHO "embulk-output-oracle test-merge2 FAILED!")
