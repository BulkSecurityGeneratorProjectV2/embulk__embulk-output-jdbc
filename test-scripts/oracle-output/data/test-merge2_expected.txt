
        ID CODE     VALUE                                                       
---------- -------- ----------------                                            
         2 BBBB     bbb                                                         
         3 AAAA     xxx                                                         
         4 YYYY     yyy                                                         

